docs/graphene-workload-tutorial/workload/tests/test_work_orders.json.template

```diff
@@ -0,0 +1,26 @@
+{
+  "workOrders": [
+    {
+      "workloadId":"python-{{ my_workload }}",
+      "params": "151234321"
```

Comment:
Can we keep params value as empty in the template as this is specific to workload

Reply:
Done